# get_touch_act_1.msg

int32[] finger_ids    
int32[] touch_values   
string[] finger_names   

